src/App.js

```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
```

Comment:
- [ ] According to the project requirements the app component should be a class component, Kindly update it.

Reply:
This issue is well resolved! ✔️